### PR TITLE
added compute_stats and get_transit_model to API list

### PR DESCRIPTION
**Pull request by vvilhelmus**
Committer: vvilhelmus <vvilhelmus@protonmail.com>

(first time doing this, my apologies in advance if I am doing it wrong)

### CODE REVIEW

**Review by orionlee — CHANGES_REQUESTED** [review]
Thanks for the submission! 

The 2 methods are specific to BLS Periodogram only. I'd add them to the end of the Methods section as.


```
  BoxLeastSquaresPeriodogram.compute_stats
  BoxLeastSquaresPeriodogram.get_transit_model
```

You can build the documentation locally to see how it looks.

https://docs.lightkurve.org/about/documentation.html

**Comment by vvilhelmus** [comment]
Hello Sam,
To be honest I was very puzzled with your reply-post and the sudden revival of this ignored thread I started last year, but then I saw that the new 2.1.1 version of LightKurve has been released only a couple of weeks from now. Would you like me to give this fresh LightKurve version 2.1.1 a spin and look for bugs perhaps?
I will definitely give editing the LightKurve documentation another shot, since I currently do have free time on my hands. I remember from last year I got stuck, when I was trying to test the changes to see if my edits resulted into the correct presentation on the webpage. I am still not sure how I can do this offline, but I will look into it again and try to learn it as I go along.
Thank you very much for the hint btw. Is this 'BLS Periodogram only"-thing something new that has come with LightKurve version 2.1, because I don't recollect encountering it last year? I probably most likely have overlooked it last year though.
Anyways, I will see what I can learn and do this time.

**Comment by barentsen** [comment]
Thanks @vvilhelmus!

You should (🤞) be able to build the docs using the guidelines posted here: https://docs.lightkurve.org/about/documentation.html

**Comment by vvilhelmus** [comment]
Okay, I finally did it, Sam @orionlee & Geert @barentsen ! :smile: 
My apologies for the waiting time, there was a lot of procrastination in this project and not everything went so smoothly as I wanted it to go. Next to installing extra stuff like Pandoc, I had to also install Makefile in my Manjaro OS to make the 'make' commands work.
Then I tried to edit the website with the BlueGriffon editor program, but that didn't work out for me and even blocked Jupyter Notebook from working, so I used the BlueFish editor program instead. Later on I found out that you guys are using the Sphinx program and perhaps I should have used that instead maybe? But I was already too far in, so I dunno... :shrug: 
Anyway, I ended up editing nearly 200 html files with BlueFish editor just for adding the two entries in the API documentation and they can all be found in the below zipped html-folder with the respective directory structure intact, so unzipping it in the original lightkurve-docs folder should automatically replace all the necessary files.
[html.zip](https://github.com/lightkurve/lightkurve/files/8727241/html.zip)
I don't think I have write permission to LightKurve’s GitHub repository, hence why I did it this way. Please let me know if you find any problems?

Lastly, I have to say, I have gained (next to some mediocre website editing skills) a newfound respect for you LightKurve developers for all the work you have put into this program and this website, thank you so much! :+1:

**Comment by orionlee** [comment]
@vvilhelmus It sounds like you've spent lots of time on this documentation change!

But I'm afraid there is a misunderstanding, the html files are generated, and are not meant to be edited directly. Again, see [building documentation](https://docs.lightkurve.org/about/documentation.html) for specifics.


The initial commit of changing `docs/source/reference/periodogram.rst` is on the right path. It's just the specific addition is not correct. Something like the following would need to be added instead:

```
  BoxLeastSquaresPeriodogram.compute_stats
  BoxLeastSquaresPeriodogram.get_transit_model
```

Once changes to `docs/source/reference/periodogram.rst` is done,  you can rebuild the documentation to verify the change has the desired effect.

**Comment by vvilhelmus** [comment]
Sam @orionlee , yes, I spent roughly about ~40 hours of my free time scattered over the last couple of weeks learning & figuring out how things work.
You keep linking that webpage, but all it shows is how to download the files, and (with permission) how to upload the files. Here is a screenshot of how it looks from my side:
Screenshot from 2022-05-20 10-52-34
I do not know how the website looks from your side, but from my side there is no mention or step of how to start the file edits.
Since editing with simple text editors is apparently wrong, what would then be the proper way to edit the files?

**Comment by orionlee** [comment]
Somehow we are not communicating. Let me try again.
1. One adds the following to `docs/source/reference/periodogram.rst`:
```
  BoxLeastSquaresPeriodogram.compute_stats
  BoxLeastSquaresPeriodogram.get_transit_model
```
2. To ensure the changes is correct, one then [builds the documentation](https://docs.lightkurve.org/about/documentation.html) locally to verify the resulting html.

**Comment by orionlee** [comment]
> So if I understand this correctly: when I add a Jupyter Notebook to one of the Tutorials subfolders, will the 'make html' command turn that python notebook into a complete webpage then?

That's the basic idea. One might need to edit another page or `.py` file to tell the system to include and/or link the notebook. I myself is not very proficient in the doc build system either.

Closing this in favor of PR #1210
